airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -174,11 +174,11 @@ def path(self, **kwargs) -> str:
     def request_params(self, stream_state=None, **kwargs):
         stream_state = stream_state or {}
         params = super().request_params(stream_state=stream_state, **kwargs)
-        params["fields"] = ["attachment", "issuelinks", "security", "issuetype", "created"]
+        params["fields"] = ["attachment", "issuelinks", "security", "issuetype", "updated"]
```

Comment:
I see neither updated nor created fields in the Issues schema file. how do we get it?
also why are we specifying fields here? I see none of them in schema file for Issues

Reply:
@vitaliizazmic

Reply:
@keu "created" it is a property of field property, which is presented in schema file. Unfortunately [Jira API documentation](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-search/#api-rest-api-3-search-get-responses) doesn't describe all properties of field and schema for field property is unclear. I'm not sure if "updated" is presented there.
Jira Query Language (JQL) is used for getting Issues list. It returns a big amount of fields by default, including related streams. To reduce response size field parameter is used. Which accepts a list of fields to return for each Issue. In this list are presented all fields, which can not be implemented as separated stream, and cursor.

Reply:
<img width="868" alt="Screenshot 2021-07-12 at 14 51 57" src="https://user-images.githubusercontent.com/1619536/125283054-0a69e680-e2e6-11eb-9340-e9eb053c4c01.png">
@vitaliizazmic maybe we are talking about different schema, but I don't see those fields here

Reply:
@keu yes, there is no "created" field in the schema file. There is filed "fields", which is object. I'm not sure why properties of "fields" are missed. I think, it caused by lack of documentation. I don't remeber if there is "updated" property of "fields".

Reply:
It is there see the last comment to @vitaliizazmic question

Reply:
@po3na4skld if you declaring the cursor field you must specify this field in the schema. As I can see the field is not there.

Reply:
added @keu